packages/gestalt/src/Image.js

```diff
@@ -144,6 +147,7 @@ export default class Image extends PureComponent<Props> {
         <img
           alt={alt}
           className={styles.img}
+          crossOrigin={crossOrigin}
```

Comment:
From [the MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-crossorigin), it looks like providing the attribute at all changes the behavior. So we should only apply the attribute if the prop is defined

Reply:
Yeah I tests this, and the property does not appear on the img when it's not defined, you can inspect the docs and see that crossorigin only gets added when the prop is defined

Reply:
Nice!